datasets/qa_org/users/perms_test_user.json

```diff
@@ -0,0 +1,13 @@
+{
+    "FirstName": "PermsTestingUser",
+    "LastName": "ForGiftEntry",
+    "Email": "testinggeperms@sfdx.org",
+    "Alias": "permtest",
+    "TimeZoneSidKey": "America/Los_Angeles",
+    "LocaleSidKey": "en_US",
+    "EmailEncodingKey": "UTF-8",
+    "LanguageLocaleKey": "en_US",
+    "profileName": "sysadmin_test",
+    "permsets": ["QA", "Gift_Entry_Perms"],
```

Comment:
Does the Gift_entry_perms always exist on any org?  comes by default?

Reply:
This PR adds it for the qa_org, qa_org_namespaced, and regression_org flows.